Mathlib/Algebra/GroupWithZero/Indicator.lean

```diff
@@ -40,6 +40,12 @@ lemma indicator_mul_right (s : Set ι) (f g : ι → M₀) :
   · rfl
   · rw [mul_zero]
 
+lemma indicator_mul_const (s : Set ι) (f : ι → M₀) (a : M₀) (i : ι) :
```

Comment:
Hmm, this file looks inconsistent on whether the statements are applied or not - I'd guess this one should be named `indicator_mul_const_apply`. What do you think?

Reply:
Maybe, but these new lemmas exactly match `indicator_mul_left` and `indicator_mul_right`, which I would rather not rename here because they are used in a few other files, unless you insist?

Reply:
Yeah that's fair, let's avoid it in this PR.